### PR TITLE
Search for images of people on TMDB

### PR DESCRIPTION
**Pull request by nrdwr-timmmi**
Now you can search for images of people on TMDB

### CODE REVIEW

**Comment by dinghino** [comment]
seems you can search tv shows too, actually.

File: index.js [hunk 1]
```diff
@@ -173,7 +183,7 @@ var getOmdb = function(id, type) {
     });
 };
 
-var getTmdb = function(id) {
+var getTmdb = function(id,type) {
```
Comment (vankasteelj) [inline index.js]: you should add somewhere 

`if (!type) type = 'movie';`

to allow backwards compatibility

**Review by vankasteelj — CHANGES_REQUESTED** [review]
thanks, a review below

**Comment by nrdwr-timmmi** [comment]
I have update the file. Can you check again?

**Comment by dinghino** [comment]
I'll try this out as soon as possible. I'm using it on a back end app that have a custom plugin to get shows' images from tmdb, so it may take a little to handle the refactoring.
For now thanks for the update :)

**Comment by vankasteelj** [comment]
It shouldn't affect the way it used to work. I don't really know since I moved to https://github.com/vankasteelj/mdata

**Pull request merged**